enterprise/server/remote_execution/execution_server/execution_server.go

```diff
@@ -324,15 +324,7 @@ func (s *ExecutionServer) operationFromExecution(ctx context.Context, te *tables
 }
 
 func (s *ExecutionServer) readProtoFromCAS(ctx context.Context, d *digest.InstanceNameDigest, msg proto.Message) error {
```

Comment:
nit: Should we remove this function and call cachetools.ReadProtoFromCAS directly?
There are only two calls that need to be updated.

Reply:
Done